hack/verify-flags/known-flags.txt

```diff
@@ -149,6 +149,7 @@ input-dirs
 insecure-bind-address
 insecure-port
 insecure-skip-tls-verify
+instance-name-prefix
```

Comment:
Do you get an error without this? I would not expect verify-flags to include test flags...